samples/bot-all-cards/nodejs/.localConfigs.testTool

```diff
@@ -0,0 +1,4 @@
+# A gitignored place holder file for local runtime configurations when debug in test tool
+BOT_ID=
```

Comment:
.localConfigs.testTool file needed or not?

Reply:
yes this file is needed